release-cloudbuild-nightly.yaml

```diff
@@ -0,0 +1,138 @@
+steps:
```

Comment:
I'm wondering in general whether we need to release/build Docker images for all python versions for these nightly builds?

Is the idea behind it to make sure our release workflows run without issues, or are we expecting people to actively use these releases in environments in which they're restricted to a specific python version?

Reply:
The main use case is internal, tbh, for demos. But it is also part of our reworking of the test process in general. I'd be fine only pushing one tagged image for now @ Python 3.10 or something. We can always add more if people ask for them.